### DISCUSSION TITLE
[Feature Request] B站动态引入了充电专属动态，希望可以屏蔽掉

### DISCUSSION THREAD

**Discussion by GalaxySnail** [the1812/Bilibili-Evolved#4033] [功能建议 / Ideas]
例如：https://t.bilibili.com/767252932219371541 ，出现在动态页占好大一块版面，如果可以屏蔽或者折叠掉就好了。

![](https://user-images.githubusercontent.com/53637421/221510971-a2916dd8-310d-443c-b161-65ff61e31d40.png)

**Comment by archerzhoujj** [comment]
非常需要

**Comment by LJason77** [comment]
如果有 uBlock Origin ，可以添加如下规则：

```
t.bilibili.com##div.bili-dyn-list__item:has(div:has-text(/充电专属/i))
t.bilibili.com##div.bili-dyn-list__item:has(div.bili-dyn-upower-lottery)
t.bilibili.com##div.bili-dyn-list__item:has(div.bili-dyn-card-reserve)
t.bilibili.com##div.bili-dyn-list__item:has(div.dyn-blocked-mask.feed-item)
```

我不是前端，魔改其他规则实现的。
如果有懂的大佬欢迎讨论。

**Comment by archerzhoujj** [comment]
找到了一个油猴脚本https://greasyfork.org/zh-CN/scripts/455969-b%E7%AB%99%E5%8A%A8%E6%80%81%E9%A1%B5%E9%9D%A2%E4%BD%BF%E7%94%A8%E4%BD%93%E9%AA%8C%E5%A2%9E%E5%BC%BA
